operator/.codegen.yaml

```diff
@@ -1,2 +1,7 @@
+schemapatch:
+  requiredFeatureSets:
+  - ""
+  - "Default"
+  - "TechPreviewNoUpgrade"
 swaggerdocs:
```

Comment:
I don't really know whether this is related to that file, but I'm missing the generation for the knownTypes in the alpha1 register:

https://github.com/hasbro17/api/blob/add-etcd-backups-api/operator/v1alpha1/register.go#L34-L38

those should also include the backup types. Which are important for the test fake clients, they currently don't find them:

> 	panic: no kind is registered for the type v1alpha1.EtcdBackup in scheme "github.com/openshift/client-go/operator/clientset/versioned/fake/register.go:15"

Reply:
that's what it probably should look like to pass the tests, not sure if this would be generated

```
func addKnownTypes(scheme *runtime.Scheme) error {
	metav1.AddToGroupVersion(scheme, GroupVersion)

	scheme.AddKnownTypes(GroupVersion,
		&GenericOperatorConfig{},
		&ImageContentSourcePolicy{},
		&ImageContentSourcePolicyList{},
		&EtcdBackup{},
		&EtcdBackupList{},
	)

	return nil
}

```

Reply:
Good catch, forgot to register the actual types under the group. Still working on the other updates but pushing this change in the meantime. Hopefully they show up now with the fake clients.